File: GTTS/INL.VettingService.Models/SaveLeahyVettingHit_Param.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace INL.VettingService.Models
{
    public class SaveLeahyVettingHit_Param : ISaveLeahyVettingHit_Param
    {
        public long? LeahyVettingHitID { get; set; }
        public long? PersonsVettingID { get; set; }
        public string CaseID { get; set; }
        public byte? LeahyHitResultID { get; set; }
        public byte? LeahyHitAppliesToID { get; set; }
        public byte? ViolationTypeID { get; set; }
        public DateTime? CertDate { get; set; }
        public DateTime? ExpiresDate { get; set; }
        public string Summary { get; set; }
        public long? ModifiedByAppUserID { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.VettingService.Models/save-leahy-vetting-hit_param.ts


import { ISaveLeahyVettingHit_Param } from './isave-leahy-vetting-hit_param';

export class SaveLeahyVettingHit_Param implements ISaveLeahyVettingHit_Param {
  
	public LeahyVettingHitID?: number;
	public PersonsVettingID?: number;
	public CaseID: string = "";
	public LeahyHitResultID?: number;
	public LeahyHitAppliesToID?: number;
	public ViolationTypeID?: number;
	public CertDate?: Date;
	public ExpiresDate?: Date;
	public Summary: string = "";
  
}


